### PR TITLE
Set up file structure

### PR DESCRIPTION
**Pull request by edlsto**
#### What's this PR do?
Clears out the boilerplate code for Create React App. Sets up the file structure of components for React app.

#### Where should the reviewer start?
Look at all file structure.

### CODE REVIEW

File: src/App.test.js [hunk 1]
```diff
@@ -1,9 +0,0 @@
-import React from 'react';
```
Comment (dreardon1021) [inline src/App.test.js:1]: Should we keep this test and imports? I dont see why not.

**Review by dreardon1021 — COMMENTED** [review]
Looks good, I think we should leave the App.test.js initial test and imports as were gonna be using those anyway. Other than that lets merge

File: src/App.test.js (hunk 1 above)
Comment (edlsto) [reply src/App.test.js:1]: It's still there. I think it's showing red because I moved src/App.test.js into src/App/App.test.js. Same with App.js.

Comment (dreardon1021) [reply src/App.test.js:1]: Sounds good merging in now

**Pull request merged**